python/doc/usecases/use_case_stiffened_panel.rst

```diff
@@ -0,0 +1,207 @@
+stress component :math:`\tau_{xy}`. It leads to the relation:
+
+.. math::
+   N_{xy}=q_1+q_c
+
+where :math:`q_1` abd :math:`q_c` are the shear fluxes in the panel main surface
+and its stiffener. They are given by:
+
+.. math::
+   q_1=\tau_{xy}t_s=2G_sh_0t_s\frac{\partial^2w}{\partial x\partial y}
+
+and
+
+.. math::
+  q_c=\frac{G_ct_cp}{\ell}\left[h-2h_0+\frac{h_c}{2p}(f_1-f_2)\right]\frac{\partial^2w}{\partial x\partial y}
+
+where
+
+- :math:`G_s=\frac{E_s}{2(1+\nu_s)}` is the shear modulus of the panel main
+  surface;
+
+- :math:`\frac{\partial^2w}{\partial x\partial y}` is the torsion strain of the
+  panel main surface;
+
+- :math:`G_c=\frac{E_c}{2(1+\nu_c)}` is the shear coefficient of the stiffener;
+
+- :math:`t_c` is the thickness of the stiffener;
+
+- :math:`h_c` is the distance between the mean surfaces of the stiffener hat and
+  foot;
+
+- :math:`h=h_c+\frac{t_c+t_s}{2}` is the distance between the mean surfaces of
+  the stiffener hat and the panel main surface;
+
+- :math:`f_1` is the width of the foot of the stiffener;
+
+- :math:`f_2` is the width of the hat of the stiffener;
+
+- :math:`p` is the half-widht of the stiffener;
+
+- :math:`R` is the radius of the circular part of the stiffener;
+
+- :math:`\theta` is the angle of the circular part of the stiffener;
+
+- :math:`\ell` is the length of the stiffener flank;
+
+- :math:`d=\frac{\ell-f_2}{2}-R\theta- :math:` is the half-lenght of the straight
+  part of the flank of the stiffener;
+
+- :math:`A=\ell t_c` is the area of the section of an half-ondulation;
+
+- :math:`\bar{A}=A+pt_s+\frac{1}{2}(f_1-f_2)t_c` is the area of the section of
+  the full panel (main surface and stiffener) bounded by :math:`p`;
+
+- :math:`h_0=\frac{1}{2\bar{A}}\left[A(h_c+t_c+t_s)+\frac{1}{2}t_c(f_1-f_2)(t_c+t_s)\right]`
+  is the distance between the mean surface of the panel main surface and the
+  global geometric center of the pa,nel;
+
+It leads to:
+
+.. math::
+  N_{xy}=q_1(1+q_c/q1)=\tau_{xy}t_s\left[1+\frac{1}{4}\frac{G_ct_c}{G_st_s}\frac{\left(2p(h-2h_0)-h_c(f_1-f_2)\right)}{h_0\ell}\right]
+
+and finally, :math:`(N_{xy})_{cr}` is given by:
+
+.. math::
+  (N_{xy})_{cr}=\bigg[5.35 + 4\left(\frac{b_0}{a}\right)^2\bigg]\bigg[\frac{\pi^2}{b_0^2}\frac{E_st_s^3}{12(1-\nu_s^2)}\bigg]\bigg[1+\frac{1}{4}\frac{G_ct_c}{G_st_s}\frac{(2p(h-2h_0)-h_c(f_1-f_2))}{h_0\ell}\bigg]
+
+For industrial constraints, the stiffener and the main surface are cut in the
+same metal sheet, so :math:`E_c=E_s=E`, :math:`\nu_c=\nu_s=\nu`, :math:`t_c=t_s=t`.
+The final expression of the critical shear force is then:
+
+.. math::
+  (N_{xy})_{cr}=\bigg[5.35 + 4\left(\frac{b_0}{a}\right)^2\bigg]\bigg[\frac{\pi^2}{b_0^2}\frac{Et^3}{12(1-\nu^2)}\bigg]\bigg[1+\frac{1}{4}\frac{(2p(h-2h_0)-h_c(f_1-f_2))}{h_0\ell}\bigg]
+
+with:
+
+- :math:`A=\ell t`
+
+- :math:`\bar{A}=A+t\left(p+\frac{f_1-f_2}{2}\right)`
+
+- :math:`h_0=\frac{1}{2\bar{A}}\left[A(h_c+2t)+t^2(f_1-f_2)\right]`
+
+- :math:`h=h_c+t`
+
```

Comment:
At this point, a classical ABC study defines a quantity of interest (QoI) e.g. the mean or the probability of exceeding a threshold. According to the example, that is the latter here. I suggest to introduce this topic here, independently of the the script.

Reply:
I don't really understand your point. Do you mean I have to write the description of a minimal study here?  There is no such description in other test-cases (viscous free fall, fire satellite, vertical deflection, wing weight and probably others) and I find it a good practice. It is up to the user to imagine an application, and the examples are here to give her/him ideas!

Reply:
Please read the [use_case_cantilever_beam.html](http://openturns.github.io/openturns/latest/usecases/use_case_cantilever_beam.html). This example is often used as a reliability problem: a threshold is suggested.  There are other similar examples (e.g. [use_case_beam.html](http://openturns.github.io/openturns/latest/usecases/use_case_beam.html) and [use_case_flood_model.html](http://openturns.github.io/openturns/latest/usecases/use_case_flood_model.html)). Of course, the use case is not limited to the QoI that we suggest, but this gives a starting point for the user. You might suggest a similar idea in this particular study.

Reply:
IMO there is a confusion between the presentation of a use-case and the examples related to a given use-case. It is particularly clear in the http://openturns.github.io/openturns/latest/usecases/use_case_flood_model.html use-case, where the presentation of the use-case is followed by a study, which IMO is not the right place. Everything after "Moreover, we assume that the input random variables are independent." should be put in an example or two: probability of failure, sensitivity analysis, calibration, it is way too much to ask for the definition of a use-case.

Here are the use-cases without any reference to an UQ application:
  * [The Ishigami function](https://openturns.github.io/openturns/latest/usecases/use_case_ishigami.html)
  * [The logistic model](https://openturns.github.io/openturns/latest/usecases/use_case_logistic.html)
  * [The Ackley test case](https://openturns.github.io/openturns/latest/usecases/use_case_ackley.html)
  * [The Branin test case](https://openturns.github.io/openturns/latest/usecases/use_case_branin.html)
  * [The Chaboche mechanical model](https://openturns.github.io/openturns/latest/usecases/use_case_chaboche.html)
  * [Vertical deflection of a tube](https://openturns.github.io/openturns/latest/usecases/use_case_deflection_tube.html)
  * [A viscous free fall example](https://openturns.github.io/openturns/latest/usecases/use_case_viscous_fall.html)
  * [The Fire Satellite model](https://openturns.github.io/openturns/latest/usecases/use_case_fireSatellite.html)
  * [The Wing weight function](https://openturns.github.io/openturns/latest/usecases/use_case_wingweight.html)
  * [Coles datasets](https://openturns.github.io/openturns/latest/usecases/coles.html)
  * [The Linthurst data set](https://openturns.github.io/openturns/latest/usecases/use_case_linthurst.html)

so 11 over 15 use-cases.

BTW I have quite a lot of other tasks (including the review of your PRs) so I don't plan to spend much more time on this PR. We can discuss about the perimeter of what a use-case should be in a Technical Committee and then open an issue to fix either the 11 previous use-cases or the 4 others.